### PR TITLE
updated build tools readme

### PR DESCRIPTION
**Pull request by hiko1129**
## the order of the explanation is wrong.
I think the order of the explanation is wrong.
`flutter packages pub global activate webdev` should be run from the root of the flutter project.

### CODE REVIEW

**Comment by hiko1129** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -119,7 +105,22 @@ and then you may use the `webdev` command directly from your terminal.
...
-2. Update packages.
+2. Install [`webdev` package](https://pub.dartlang.org/packages/webdev)
+
+    webdev package,
+    which provides the build tools for Flutter for web, run the following:
+
+    ```console
+    $ pub global activate webdev
+    # or
+    $ flutter packages pub global activate webde
```
Comment (geminiyellow) [inline README.md]: opps, you need `v`

```diff
- $ flutter packages pub global activate webde
+ $ flutter packages pub global activate webdev
```

**Comment by geminiyellow** [comment]
and after do this, there is another error:

```
/hello_world> flutter packages upgrade
! flutter_web 0.0.0 from path ../../packages/flutter_web                
! flutter_web_ui 0.0.0 from path ../../packages/flutter_web_ui          
Running "flutter packages upgrade" in hello_world...               15.9s

/hello_world> webdev serve
/Users/dev/Documents/Source/flutter/.pub-cache/bin/webdev: line 7: dart: command not found
```

File: README.md (hunk 1 above)
Comment (hiko1129) [reply README.md]: Thank you.
I fixed it.

**Comment by hiko1129** [comment]
I signed it!

**Comment by geminiyellow** [comment]
I signed it!

**Comment by hiko1129** [comment]
Am I wrong?

**Comment by hiko1129** [comment]
It worked.

**Comment by yjbanov** [comment]
@kevmoo would you like to review this?

**Comment by kevmoo** [comment]
We need to fix up the global activate bits w/ the Flutter SDK. I'll take care of this in another commit.

Thanks!

**Pull request closed without merging**